layouts/speaker/single.html

```diff
@@ -72,6 +72,30 @@ <h2 class="speaker-page">{{ .Title }}</h2>
               {{- end -}} <!-- end if eq $talk_slug $event_slug -->
             {{- end -}} <!-- end range where $.Site.Pages "Type" "speaker" -->
           </ul>
+    <br>
+      <h3>Workshops by {{ .Title }}</h3>
```

Comment:
Just move the headline down to the part where you display it only if they match, etc.